fem/pfespace.hpp

```diff
@@ -428,6 +428,33 @@ class ParFiniteElementSpace : public FiniteElementSpace
       old_dof_offsets.DeleteAll();
    }
 
+   /** @brief Return the true-dof to true-dof update operator corresponding to
```

Comment:
Could a small unit test be added, for the various types of updates?

Reply:
I'll have to think about, I'm not sure what will be a good way to check correctness of the result(s).

Reply:
I added a unit test in `tests/unit/fem/test_update_operators.cpp` with b3d1e872. Does it look okay?

Reply:
Thanks, it looks good!